tests/displayed_attributes_tests.ts

```diff
@@ -145,3 +147,50 @@ describe.each([{ client: anonymousClient, permission: 'No' }])(
     })
   }
 )
+
+test(`Get request should not add double slash nor a trailing slash`, async () => {
+  try {
+    const res = await badHostClient.index(index.uid).getDisplayedAttributes()
+    expect(res).toBe(undefined) // Left here to trigger failed test if error is not thrown
+  } catch (e) {
+    expect(e.message).toMatch(
+      `${BAD_HOST}/indexes/movies_test/settings/displayed-attributes`
+    )
+    expect(e.message).not.toMatch(
+      `${BAD_HOST}/indexes/movies_test/settings/displayed-attributes/`
+    )
```

Comment:
are these lines useful if the first test pass?
First test =
```
expect(e.message).toMatch(...)
```

Reply:
Yes! Because the first one is only "contained" and not an exact match. 
So if my error is
"Could not connect to http://localhost:7700/indexes/happynewyear" 
The first tests will work, because the string contains the sub-string `http://localhost:7700/indexes`.
If I make the second tests I ensure: 
1. no trailing parameters
2. Consistency between every request made, which are without a trailing slash

Reply:
Got it!